src/open_inwoner/mijn_afval/views.py

```diff
@@ -1,11 +1,30 @@
-from django.shortcuts import render
+from django.urls import reverse
+from django.utils.functional import cached_property
+from django.views.generic import TemplateView
+
+from view_breadcrumbs import BaseBreadcrumbMixin
+
+from open_inwoner.utils.views import CommonPageMixin
 
 from .clients import AfvalApiClient
 
 
-def afval_view(request):
-    client = AfvalApiClient(base_url="")
+class AfvalView(BaseBreadcrumbMixin, CommonPageMixin, TemplateView):
```

Comment:
The view is already implemented here: https://github.com/maykinmedia/open-inwoner/pull/2079. I've added the breadcrumbs. Let me know if you're missing anything else, I'll fix it in that PR so we avoid merge conflicts.